osu.Game/Screens/Play/Player.cs

```diff
@@ -1159,7 +1159,10 @@ protected virtual Task ImportScore(Score score)
         /// </summary>
         /// <param name="score">The <see cref="ScoreInfo"/> to be displayed in the results screen.</param>
         /// <returns>The <see cref="ResultsScreen"/>.</returns>
-        protected virtual ResultsScreen CreateResults(ScoreInfo score) => new SoloResultsScreen(score, true);
+        protected virtual ResultsScreen CreateResults(ScoreInfo score) => new SoloResultsScreen(score, true)
+        {
+            ShowUserStatistics = true
```

Comment:
Interesting. I thought last time we looked into `init` parameters it didn't allow usage in object initialisers. This is definitely something we'll want to make use of more.